src/Map/ProspectorOverlayLayer.cs

```diff
@@ -266,10 +359,14 @@ private string GetArgumentPositionAsString(int position)
         #endregion
 
         #region Texture
-        private LoadedTexture GenerateOverlayTexture()
+        private LoadedTexture GenerateOverlayTexture(RelativeDensity? relativeDensity)
         {
             var colorTexture = new LoadedTexture(_clientApi, 0, _chunksize, _chunksize);
-            var colorArray = Enumerable.Repeat(_config.TextureColor.RGBA, _chunksize * _chunksize).ToArray();
+            int[] colorArray;
+            if (_config.MapMode == 1)
```

Comment:
Would it be possible to make the MapMode an Enum ?  Either by changing it directly or by making a getter that translates the 0 1 to the proper things ?